include/fastrtps/rtps/builtin/liveliness/WLP.h

```diff
@@ -205,49 +205,47 @@ class WLP
     LivelinessManager* sub_liveliness_manager_;
 
     /**
-     * @brief A method invoked by pub_liveliness_manager_ to inform that a writer lost liveliness
+     * @brief A method invoked by pub_liveliness_manager_ to inform that a writer changed its liveliness
      * @param writer The writer losing liveliness
      * @param kind The liveliness kind
      * @param lease_duration The liveliness lease duration
+     * @param alive_change The change in the alive count
+     * @param not_alive_change The change in the not alive count
      */
-    void pub_liveliness_lost(
+    void pub_liveliness_changed(
             const GUID_t& writer,
             const LivelinessQosPolicyKind& kind,
-            const Duration_t& lease_duration);
+            const Duration_t& lease_duration,
+            int alive_change,
```

Comment:
This parameter and the next one should be int32_t, right?

Reply:
Yes, it is changed now